test/OpenTelemetry.Exporter.Geneva.Tests/GenevaLogExporterTests.cs

```diff
@@ -1132,6 +1132,133 @@ public void SerializationTestForEventName(EventNameExportMode eventNameExportMod
         }
     }
 
+    [Theory]
+    [InlineData(false, false, "Custom name")]
+    [InlineData(false, false, "")]
+    [InlineData(false, false, null)]
+    [InlineData(false, false, 5)]
+    [InlineData(true, false, "Custom name")]
+    [InlineData(true, true, "Custom name")]
```

Comment:
minor: add a case for (true, true, "13245") to exercise the non-string validation (for the case that name doesn't follow PartB convention) path?

Reply:
I assume you mean 12345 (not a string). Just added it

Reply:
> I assume you mean 12345 (not a string). 

you're right! : ) thank you